microsoftgraph/microsoft-graph-explorer-v4
# Fix: support different content types

## Overview

Partially fixes #626 
Now successfully supports HTML responses.

### Demo

Request: https://graph.microsoft.com/v1.0/me/onenote/pages/{pages-id}/content

Before
![image](https://user-images.githubusercontent.com/58787602/105984053-aabaae80-60aa-11eb-8ac1-bf6327429cfe.png)

After
![image](https://user-images.githubusercontent.com/58787602/105984253-e5244b80-60aa-11eb-816d-56412f91f59a.png)



### Notes

I can not conclusively say that it fixes #626 since I have not been able to test any call that returns an `edm.stream` response. It errors out with a 403, which is weird since I already have the required permission consented to. 

![image](https://user-images.githubusercontent.com/58787602/105984677-87443380-60ab-11eb-987b-a8f0d5936575.png)


## Testing Instructions

* Run this query: https://graph.microsoft.com/v1.0/me/onenote/pages/{pages-id}/content after replacing the `{pages-id}` token with the appropriate page id
* Notice the HTML response is currently visible
